docs/en/query_language/functions/array_functions.md

```diff
@@ -994,11 +994,13 @@ arrayZip(arr1, arr2, ..., arrN)
 
 **Parameters**
 
-`arr` — Any number of [array](../../data_types/array.md) type columns to combine.
+- `arr` — Any number of columns to combine. [Array](../../data_types/array.md).
```

Comment:
```suggestion
- `arrN` — [Array](../../data_types/array.md).
```

The function can take any number of arrays. All the arrays must have the same length.